src/10_Introduction/AMP_for_E-Commerce_Getting_Started.html

```diff
@@ -98,7 +123,64 @@
     * **Star ratings:** [here](/advanced/star_rating/) is a sample demonstrating how to implement a star rating system in AMP.
     * **Call tracking:** use [amp-call-tracking](/components/amp-call-tracking) to track calls initiated from your AMPs.
     * **Comments / reviews:** comment or review submission can be implemented using [amp-form](/components/amp-form). Combine with [amp-access](/components/amp-access/) if a user login is required. [Here](/samples_templates/comment_section/preview/) is a sample combining both.
-
+ -->
+ <!-- ## Scroll to top
+
+ For helping users browsing on a long list of items,
+ you can implement a `scroll to top` button by using the `scrollTo` action, [amp-position-observer](https://www.ampproject.org/docs/reference/components/amp-position-observer) and [amp-animation](https://www.ampproject.org/docs/reference/components/amp-animation).
+ Find more about actions [here](https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md).
+
+ We use 2 `amp-animation` elements to trigger the visibility of the button.
+ The first one is for making the button visible...
+ -->
+ <amp-animation id="showAnim" layout="nodisplay">
+   <script type="application/json">
+     {
+       "duration": "200ms",
+         "fill": "both",
+           "iterations": "1",
+             "direction": "alternate",
+               "animations": [
+                 {
+                   "selector": "#scrollToTopButton",
+                   "keyframes": [
+                     { "opacity": "1", "visibility": "visible" }
+                   ]
+                 }
+               ]
+     }
+   </script>
+ </amp-animation>
+ <!-- ... and the second one is for adding the button.-->
+ <amp-animation id="hideAnim" layout="nodisplay">
+  <script type="application/json">
+    {
+      "duration": "200ms",
```

Comment:
spacing is off (my bad from the sample, codepen does weird formatting)

Reply:
done!